--- AppDelegate.cpp	2014-12-11 10:53:24.000000000 -0800
+++ AppDelegate.cpp	2014-12-11 10:53:24.000000000 -0800
@@ -13,6 +13,12 @@
 #include "ide-support/RuntimeLuaImpl.h"
 #endif

+#ifdef SDKBOX_ENABLED
+#include "Plugin$(SDKBOX_PLUGIN_NAME)Lua.hpp"
+#include "Plugin$(SDKBOX_PLUGIN_NAME)LuaHelper.h" #stripifdef NO_LUA_HELPER
+#endif
+
 using namespace CocosDenshion;

 USING_NS_CC;
@@ -48,6 +54,10 @@
 // don't modify or remove this function
 static int register_all_packages()
 {
+#ifdef SDKBOX_ENABLED
+    register_all_Plugin$(SDKBOX_PLUGIN_NAME)Lua(LuaEngine::getInstance()->getLuaStack()->getLuaState());
+    register_all_Plugin$(SDKBOX_PLUGIN_NAME)Lua_helper(LuaEngine::getInstance()->getLuaStack()->getLuaState()); #stripifdef NO_LUA_HELPER
+#endif
     return 0; //flag for packages manager
 }
